lib/rodzilla/web_service.rb

```diff
@@ -60,12 +60,12 @@ def users
 
     def bugzilla_resource(resource)
       raise Rodzilla::Error::ResourceNotFoundError, "Error: Rodzilla::Resource::#{resource} does not exist." unless Rodzilla::Resource.constants.include?(resource.to_sym)
-      @resource = Object.module_eval("Rodzilla::Resource::#{resource}").new(@base_url, @username, @password, @format)
+      @resource = Object.module_eval("Rodzilla::Resource::#{resource}").new(@base_url, @username, @password, @token, @format)
```

Comment:
I think the token goes *after* format, looking at 

https://github.com/evgeni/rodzilla/blob/8d507527d71abec36ea964a33c197d615bb10ab7/lib/rodzilla/resource/base.rb#L6

Reply:
weird... cuz this appeared to work... testing

Reply:
So, i swapped them to be "format, token", and it errored out.  And the above referenced code is there:

```
head -n 20 /home/pcreech/.rvm/gems/ruby-2.7.2/bundler/gems/rodzilla-8d507527d71a/lib/rodzilla/resource/base.rb
module Rodzilla
  module Resource
    class Base
      attr_accessor :base_url, :username, :password, :service, :format, :token

      def initialize(base_url, username, password, format=:json, token=nil)
        @base_url = base_url
        @username = username
        @password = password
        @format = format
        @token = token

        setup_service(format)
      end

      def rpc_call(rpc_method_name, params={}, http_method=:post)
        service.send_request!( get_resource_rpc_method_name(rpc_method_name), params, http_method)
      end

      # Makes a GET request to the service endpoint and checks if the 

```

Reply:
And i put them in order based on this change, which, in retrospect is weird (at the moment):

https://github.com/pcreech/rodzilla/blob/token/lib/rodzilla/json_rpc/service.rb#L11